### ISSUE TITLE
El traje nuevo del emperador: Black square floating in a corner

### ISSUE THREAD

**Issue by RafaelTrepaUnCarballo** [otya128/winevdm#1210]
**Describe the bug**
A black square with a white line (or the number "1") appears in the corner of the screen.

**Expected behavior**
No square.

**Screenshots**
![imagen](https://user-images.githubusercontent.com/16591467/195389903-cd7f3002-646f-4edc-99db-48bc16d6fbcd.png)

**Environment (please complete the following information):**
 Windows 10 Pro 64-bit (10.0, Build 19044) (19041.vb_release.191206-1406)

**Additional context**
Trace file from a slightly older build:
https://github.com/otya128/winevdm/files/9762991/trace.zip

**Comment by cracyc** [comment]
I don't think this is a bug but an artifact of the game being designed to work at 640x480 resolution.  It's impossible to know for sure without testing it directly though.

**Comment by RafaelTrepaUnCarballo** [comment]
> It's impossible to know for sure without testing it directly though.

I offer you a link to the ISO:
https://drive.google.com/file/d/16z8piAeGTg1a36F-BG5e0woaPleTsgwj/view

You have to manually execute the w31 installer, by default it autoexecutes the w95 one.

**Comment by cracyc** [comment]
I couldn't get that problem to happen.  Are you sure it's not caused by something other than winevdm?
